README.md

```diff
@@ -95,11 +95,13 @@ Usage
   * as a node attribute via an `environment` or `role`, or
   * as a node attribute by declaring it in another cookbook at a higher precedence level, or
   * in the node `run_state` by setting `node.run_state['datadog']['api_key']` in another cookbook preceding `datadog`'s recipes in the run_list. This approach has the benefit of not storing the credential in clear text on the Chef Server.
-3. Create an 'application key' for `chef_handler` [here](https://app.datadoghq.com/account/settings#api), and add it as a node attribute or in the run state, as in Step #2.
+3. If you require a private gem server to install the chef-handler-datadog gem, set a node attribute:
+  * e.g. ```default['datadog']['gem_server'] = 'http://custom.gem_server.org'```
```

Comment:
This sounds like quite a specific use case. As such I'm not sure it should be listed in the `usage` section of the readme (I'd like to keep this section brief and only describe the required attributes here). What do you think about adding this documentation in the default attributes file?